aspnetcore/fundamentals/error-handling.md

```diff
@@ -77,70 +57,57 @@ public IActionResult Error()
 
 ## Configure status code pages
 
-By default, an app doesn't provide a rich status code page for HTTP status codes, such as *404 Not Found*. To provide status code pages, use Status Code Pages Middleware.
-
-::: moniker range=">= aspnetcore-2.1"
+By default, an app doesn't provide a rich status code page for HTTP status codes, such as *404 - Not Found*. To provide status code pages, use Status Code Pages Middleware.
```

Comment:
"doesn't provide rich" implies "does provide non-rich" -- what's the difference between rich and non-rich pages?